servos: [{id: 9,joint_name: torso_joint, velocity_mode: true,use_init_vel: false,interface: velocityInterface ,protocol_version: 1.0, torque: 1, read_pos: true, read_vel: true, read_eff: false, write_pos: false, write_vel: true}]



device: "/dev/ttyUSB0"
serial_timeout_ms: 1000000000
device_baudrate: 57600
init_speed_protocol1: 1.0
init_speed_protocol2: 0.3


